### PR TITLE
Fn3 address book

### PR DESCRIPTION
**Pull request by abarth**
@Hixie

### CODE REVIEW

File: examples/hello_world/lib/main.dart [hunk 1]
```diff
@@ -2,14 +2,6 @@
...
-class HelloWorldApp extends App {
-  Widget build() {
-    return new Center(child: new Text('Hello, world!'));
-  }
-}
-
-void main() {
-  runApp(new HelloWorldApp());
-}
+void main() => runApp(new Center(child: new Text('Hello, world!')));
```
Comment (Hixie) [inline examples/hello_world/lib/main.dart]: Did you verify that this centers the widget properly? (I know we've had issues with that in the past...)

**Comment by Hixie** [comment]
LGTM assuming you verified that HelloWorld works.

File: examples/hello_world/lib/main.dart (hunk 1 above)
Comment (abarth) [inline examples/hello_world/lib/main.dart]: I didn't.  Will check.

Comment (abarth) [inline examples/hello_world/lib/main.dart]: Yep, works.

Comment (Hixie) [inline examples/hello_world/lib/main.dart]: Sweet.

**Comment by Hixie** [comment]
I love how short Hello World has become.

**Pull request merged**